### DISCUSSION TITLE
Shortcut Keys

### DISCUSSION THREAD

**Discussion by darkoverlordofdata** [helloSystem/hello#560] [Q&A]
I thought that helloSystem used to have an app for editing the shortcut keys. Sure enought, I find it referenced here:[https://hellosystem.github.io/docs/user/components/preferences/shortcuts.html](https://hellosystem.github.io/docs/user/components/preferences/shortcuts.html).

This is not a high priority, but where is this program now? Or does anyone know how to create new shortcuts? I am able to change existing ones using
```
kwriteconfig5 --file kglobalshortcutsrc ...
``` 
But how do I create a new one and point it at a custom application?

**Comment by probonopd** [comment]
Thanks for your observation. Indeed this is an open issue:
* Historically, we were using https://github.com/lxqt/lxqt-globalkeys
* Then, we switched to KWin which comes with kglobalaccel
* In order not to have 2 seemingly overlapping tools, I removed lxqt-globalkeys (and its nice and very straightforward GUI)
* Unfortunately, I haven't been able to recreate its functionality using kglobalaccel yet

Any insights appreciated. Do we have to go back to shipping 2 tools?

In any case, the documentation is outdated. I'll add a remark.

**Comment by darkoverlordofdata** [comment]
Ok. I don't see a need for two tools. I'm playing around
with kglobalshortcutsrc, maybe I can figure it out.

On Tue, Nov 14, 2023 at 3:34 PM probonopd ***@***.***> wrote:

> Thanks for your observation. Indeed this is an open issue:
>
>    - Historically, we were using https://github.com/lxqt/lxqt-globalkeys
>    - Then, we switched to KWin which comes with kglobalaccel
>    - In order not to have 2 seemingly overlapping tools, I removed
>    lxqt-globalkeys (and its nice and very straightforward GUI)
>    - Unfortunately, I haven't been able to recreate its functionality
>    using kglobalaccel yet
>
> Any insights appreciated. Do we have to go back to shipping 2 tools?
>
> In any case, the documentation is outdated. I'll add a remark.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/helloSystem/discussions/560#discussioncomment-7570791>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJUE4U2RHFHSLPE4VODCYTYEP5WZAVCNFSM6AAAAAA7KBQA6SVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TKNZQG44TC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

Bruce Davidson

**Reply by probonopd** [reply]
It would be ideal to have a GUI like the one from lxqt-globalkeys but using the kglobalshortcuts daemon instead of running yet another one.
